shop/myFlywayConfig.conf

```diff
@@ -0,0 +1,5 @@
+flyway.user=sa
+flyway.password=
+flyway.schemas=OnlineShop
+flyway.url=jdbc:h2:~/Desktop/CareerStartBackend/online-shop-SerbyRo/shop;DATABASE_TO_UPPER=FALSE
```

Comment:
Do not use paths from your computer, use just project related paths. 

Reason: anyone should be able to clone the project and start it locally without any additional files or changes to the project. If you keep this path like this I will not be able to start the project because I don't have a folder named CareerStartBackend on my Desktop.